course_content/notebooks/numpy_intro.ipynb

```diff
@@ -95,11 +95,63 @@
    "source": [
     "### Documentation\n",
     "Here is a link to the NumPy documenetation for v1.11:\n",
-    "https://docs.scipy.org/doc/numpy-1.11.0/reference/"
+    "https://docs.scipy.org/doc/numpy-1.11.0/reference/\n",
+    "\n",
+    "### Create an Array\n",
+    "\n",
+    "There are many ways to create a numpy array, such as:"
```

Comment:
I wonder whether showing more than on way would be good here. Then delegates can try out the ways they've been shown for themselves as well as going and finding still other ways of creating arrays.